foundry.toml

```diff
@@ -7,6 +7,7 @@ gas_reports = ["*"]
 optimizer_runs = 2000
 extra_output = ["storageLayout"]
 solc-version = '0.8.24'
+bytecode_hash = 'none'
```

Comment:
Have we tested whether this change possibly breaks etherscan verification? I thought etherscan used this metadata hash as part of its contract verification process

Reply:
Yeah ethesxan was able to verify the contract. The impl contract was deployed with the setup